### ISSUE TITLE
Update Rounded Buttons. 

### ISSUE THREAD

**Issue by regnerjr** [iOS-Connect/Connect#34]
Various screens through out the app use
```
        joinButton.backgroundColor = .clear
        joinButton.layer.cornerRadius = 10
        joinButton.layer.borderWidth = 2
        joinButton.layer.borderColor = self.view.tintColor.cgColor
```

It would be really cool to see this redundant code moved out of the view controller and into a Custom Sub class of UIButton.

**Comment by stellasu168** [comment]
https://github.com/iOS-Connect/Connect/pull/35

**Comment by regnerjr** [comment]
Fixed by #35
